### PR TITLE
CATROID-742 Add Placeholder names

### PR DESCRIPTION
**Pull request by Walcho1125**
*Please enter a short description of your pull request and add a reference to the Jira ticket.*
Add placeholder to any add option https://jira.catrob.at/browse/CATROID-742
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by sasalala000 — APPROVED** [review]

File: catroid/src/main/java/org/catrobat/catroid/ui/SpriteActivity.java [hunk 1]
```diff
@@ -803,9 +809,24 @@ public void handleAddUserDataButton() {
 			if (checked) {
 				alertDialog.setTitle(getString(R.string.formula_editor_list_dialog_title));
 				textWatcher.setOriginalScope(lists);
+				TextInputEditText textInputEditText = alertDialog.findViewById(R.id.input_edit_text);
```
Comment (dandrawes) [inline catroid/src/main/java/org/catrobat/catroid/ui/SpriteActivity.java:812]: Deduplicate the same code in both branches.

**Review by dandrawes — CHANGES_REQUESTED** [review]
Well done!

Please look at my small remarks and change the year range in the header to "2021" in the changed files.

The first new object is named "My actor or object" without "(1)" in the end like all other placeholders, please change this also to be consistent.

**Review by dandrawes — APPROVED** [review]
Looks good now!

File: catroid/src/androidTest/java/org/catrobat/catroid/test/io/asynctask/ProjectUnzipAndImportTaskTest.java [hunk 2]
```diff
@@ -52,9 +52,9 @@
 @RunWith(AndroidJUnit4.class)
 public class ProjectUnzipAndImportTaskTest {
 
-	private static final String AIR_FIGHT_0_5 = "Air fight 0.5";
-	private static final String FALLING_BALLS = "Falling balls";
-	private static final String AIR_FIGHT_0_5_1 = "Air fight 0.5 (1)";
+	private static final String AIR_FIGHT_0_5 = "Air fight 0.5 (1)";
```
Comment (wslany) [inline catroid/src/androidTest/java/org/catrobat/catroid/test/io/asynctask/ProjectUnzipAndImportTaskTest.java:55]: This leads to a failure in line 89 further below.

Stacktrace:
https://jenkins.catrob.at/job/Catroid/job/PR-3973/51/testReport/org.catrobat.catroid.test.io.asynctask/ProjectUnzipAndImportTaskTest/All___1___Instrumented_Unit_Tests___testUnzipAndImportSingleProject/

=>
```
java.lang.AssertionError:
Expected: a collection containing "Air fight 0.5 (1)"
but: was "Air fight 0.5"
at org.catrobat.catroid.test.io.asynctask.ProjectUnzipAndImportTaskTest.testUnzipAndImportSingleProject(ProjectUnzipAndImportTaskTest.java:89)
```

File: catroid/src/androidTest/java/org/catrobat/catroid/test/io/asynctask/ProjectUnzipAndImportTaskTest.java [hunk 3]
```diff
@@ -52,9 +52,9 @@
...
+	private static final String FALLING_BALLS = "Falling balls (1)";
```
Comment (wslany) [inline catroid/src/androidTest/java/org/catrobat/catroid/test/io/asynctask/ProjectUnzipAndImportTaskTest.java:56]: This leads to a failure in line 104 further below.

Stacktrace:
https://jenkins.catrob.at/job/Catroid/job/PR-3973/51/testReport/org.catrobat.catroid.test.io.asynctask/ProjectUnzipAndImportTaskTest/All___1___Instrumented_Unit_Tests___testUnzipAndImportMultipleProjects/
=>
```
java.lang.AssertionError:
Expected: a collection containing "Falling balls (1)"
but: was "Air fight 0.5", was "Air fight 0.5 (1)", was "Falling balls"
at org.catrobat.catroid.test.io.asynctask.ProjectUnzipAndImportTaskTest.testUnzipAndImportMultipleProjects(ProjectUnzipAndImportTaskTest.java:104)
```

**Review by wslany — CHANGES_REQUESTED** [review]
Two failing tests remain, see comments. 

Please recheck the point "Confirm that new and existing unit tests pass locally".

**Comment by Walcho1125** [comment]
> Two failing tests remain, see comments.
> 
> Please recheck the point "Confirm that new and existing unit tests pass locally".

I fixed the Problem and reran the all tests locally to make sure that all pass

**Review by wslany — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!

**Pull request merged**